_sass/components/_tiles.scss

```diff
@@ -478,7 +480,6 @@
 			object-fit: cover;
 			img {
 				border-radius: 0;
-				object-fit: contain;
 				width: 100%;
 				height: 100%;
```

Comment:
need fixed height here: 200px. 
Stretched doesn't look good: 
![image](https://user-images.githubusercontent.com/7769371/59369977-6ec8af00-8cf6-11e9-8d18-b6687f87e8ef.png)

Reply:
also a similar problem with stretched images at blog posts lists, tile height should be fixed 300px
 
![image](https://user-images.githubusercontent.com/7769371/59370810-31652100-8cf8-11e9-9318-8a58614cf6d6.png)

Reply:
can you please check Figma file for proper elements proportions?